pages/index.tsx

```diff
@@ -92,14 +91,22 @@ const Charts = ({ chartData }: { chartData: HomeChartData | null }) => {
       })) ?? [],
     [chartData]
   );
+
+  const [inflationGrouping, setInflationGrouping] = useState<Group>(Group.ALL);
   const inflationRateData = useMemo(
     () =>
-      chartData?.dayData?.slice(1)?.map((day) => ({
-        x: Number(day.dateS),
-        y: Number(day?.inflation ?? 0) / 1000000000,
-      })) ?? [],
-    [chartData]
+      (inflationGrouping === Group.YEAR
```

Comment:
Can be written as
```
chartData?.dayData?.slice(inflationGrouping ==== Group.year ? -365 : 1).map((day) => ({
            x: Number(day.dateS),
            y: Number(day?.inflation ?? 0) / 1000000000,
          })) ?? []
```

Reply:
Changed in f6620b92aa07da4db79d3f44c5972863a3db1bf9